src/compiler/membership.md

```diff
@@ -110,8 +110,8 @@ rustc-dev-guide.
 It is worth emphasizing that becoming a member of the compiler team does not necessarily imply
```

Comment:
heh seems some copy&paste slipped in. Reading again this paragraph, I think we can just trim it down to:

```suggestion
The most important criterion for elevation to compiler team member, in particular,
is **regular and consistent** participation. The most important criterion for elevation to
maintainer is **actively shaping the direction of the team or compiler**.
```

Reply:
heh, I missed that too :)